lib/livebook/teams/requests.ex

```diff
@@ -326,4 +361,12 @@ defmodule Livebook.Teams.Requests do
   defp json?(response) do
     "application/json; charset=utf-8" in Req.Response.get_header(response, "content-type")
   end
+
+  defp validate_upload_size(data) do
+    if byte_size(data) <= 20 * 1024 * 1024 do
+      :ok
+    else
+      {:error, "file size too large"}
+    end
+  end
```

Comment:
Move to schema/changeset. Change LT to have a maximum of `Teams.encrypt(:crypto.strong_rand_bytes(20 * 1024 * 1024))`.